backend/supabase/group_invites/create.sql

```diff
@@ -0,0 +1,45 @@
+drop table group_invites;
```

Comment:
This would destroy all the current data in the table, correct? I don't think it makes sense to put that in the sql file